### PR TITLE
H4C validation

### PR DESCRIPTION
**Pull request by lisaleemcb**
This contains only changes to the `validation/h4c/` directory of `hera_pipelines`. I just want to merge into main so that it plays nicely with the `makeflow_nrao` script. Thanks!

### CODE REVIEW

File: pipelines/h4c/post_processing/v6/lstbin/h4c_lstbincheck_forKaiFeng.toml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+[Options]
```
Comment (r-pascua) [inline pipelines/h4c/post_processing/v6/lstbin/h4c_lstbincheck_forKaiFeng.toml]: This looks like it shouldn't be in this PR. Could you move it to a new branch?

File: pipelines/h4c/post_processing/v6/post_lstbin_frate/task_scripts/do_PSPEC.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ spw_ranges="${5}"
 jd=$(get_jd $fn)
 int_jd=${jd:0:7}
 if [[ "$int_jd" == *"."* ]]; then
-  jd=`echo ${fn} | grep -o "[0-9]\{1,2\}.[0-9]\{5\}"`
+  jd=`echo ${fn} | grep -o "[0-9]\{1,2\}.[0-9]\{7\}"`
```
Comment (r-pascua) [inline pipelines/h4c/post_processing/v6/post_lstbin_frate/task_scripts/do_PSPEC.sh]: Should this change actually be here? It looks like a validation-specific thing.

File: pipelines/h4c/post_processing/v6/post_lstbin_frate/task_scripts/do_PSTOKES.sh [hunk 3]
```diff
@@ -21,7 +21,7 @@ pstokes="${@:4}"
 jd=$(get_jd $fn)
 int_jd=${jd:0:7}
 if [[ "$int_jd" == *"."* ]]; then
-  jd=`echo ${fn} | grep -o "[0-9]\{1,2\}.[0-9]\{5\}"`
+  jd=`echo ${fn} | grep -o "[0-9]\{1,2\}.[0-9]\{7\}"`
```
Comment (r-pascua) [inline pipelines/h4c/post_processing/v6/post_lstbin_frate/task_scripts/do_PSTOKES.sh]: Same comment as in the `do_PSPEC.sh` script.

File: pipelines/h4c/post_processing/v6/pre_lstbin/task_scripts/do_PSTOKES.sh [hunk 4]
```diff
@@ -38,17 +38,20 @@ source ${src_dir}/_common.sh
 
 
 
+
 fn="${1}"
 include_diffs="${2}"
 label="${3}"
-pstokes="${@:3}"
+pstokes="${@:4}"
```
Comment (r-pascua) [inline pipelines/h4c/post_processing/v6/pre_lstbin/task_scripts/do_PSTOKES.sh]: Is this update accurate?

File: pipelines/validation/h4c/gain_jumps/non_red_fringe_rate/calibrate_gain_jumps.toml [hunk 5]
```diff
@@ -0,0 +1,111 @@
+[Options]
```
Comment (r-pascua) [inline pipelines/validation/h4c/gain_jumps/non_red_fringe_rate/calibrate_gain_jumps.toml:1]: I don't think we're doing anything about gain jumps, right?

**Review by r-pascua — APPROVED** [review]
I think this should be fine, but it looks like there are some changes to the H4C analysis code that maybe shouldn't come along with this PR (I've added comments at each spot). I figure it should be OK to merge after a bit of tidying up.

File: pipelines/h4c/post_processing/v6/lstbin/h4c_lstbincheck_forKaiFeng.toml (hunk 1 above)
Comment (lisaleemcb) [reply pipelines/h4c/post_processing/v6/lstbin/h4c_lstbincheck_forKaiFeng.toml]: deleted!

File: pipelines/h4c/post_processing/v6/post_lstbin_frate/task_scripts/do_PSPEC.sh (hunk 2 above)
Comment (lisaleemcb) [reply pipelines/h4c/post_processing/v6/post_lstbin_frate/task_scripts/do_PSPEC.sh]: Ah, yes, these were all on the fly changes we made to make the eor sim run.  I've reverted all these changes.

**Pull request merged**